osu.Game/Overlays/Profile/Header/Components/GlobalRankDisplay.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+using osu.Framework.Extensions.LocalisationExtensions;
+using osu.Framework.Localisation;
+using osu.Game.Online.API.Requests.Responses;
+using osu.Game.Resources.Localisation.Web;
+using osu.Game.Users;
+
+namespace osu.Game.Overlays.Profile.Header.Components
+{
+    public partial class GlobalRankDisplay : ProfileValueDisplay
+    {
+        public readonly Bindable<UserStatistics?> UserStatistics = new Bindable<UserStatistics?>();
+        public readonly Bindable<APIUser?> User = new Bindable<APIUser?>();
+
+        public GlobalRankDisplay()
+            : base(true)
+        {
+            Title = UsersStrings.ShowRankGlobalSimple;
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            UserStatistics.BindValueChanged(s =>
+            {
+                Content = s.NewValue?.GlobalRank?.ToLocalisableString("\\##,##0") ?? (LocalisableString)"-";
+            }, true);
+
+            // needed as statistics doesn't populate User
```

Comment:
Can you elaborate? I don't follow what this is trying to say.

Reply:
Talking about this:

https://github.com/ppy/osu/blob/b0420f7ed9e945cb69ffd844664df0705ccc4e8f/osu.Game/Users/UserStatistics.cs#L17-L18

Seems to return null, so I have to add the `User` bindable instead of just using `UserStatistics`.

Reply:
Again, this is what happens when API models are overloaded to be used in 30 different contexts.........

I'd at least reword the comment to mention the specific class rather than vaguely mention "statistics" of unknown provenance.